commonslibrary/settings/base.py

```diff
@@ -282,12 +282,15 @@
 # yapf: disable
 CKEDITOR_CONFIGS = {
     'default': {
-        'skin': 'moono-lisa',
+        'skin': 'moono',
         'toolbar_Basic': [['Source', '-', 'Bold', 'Italic']],
         'toolbar': [{
             'name': 'basicstyles',
             'items': [
-                'Bold', 'Italic', 'Underline', 'Strike', 'Image', 'Link', 'Source',
+                'Bold', 'Italic', 'Underline', 'Strike', 'Liststyle', 'Styles', 'Image', 'Link',
```

Comment:
I'd suggest the following:

- Remove Liststyle
- Move Format to the start of the list, followed by '-', then bold etc.
- Remove Font
- Remove FontSize